alcs-frontend/src/app/shared/lots-table/lots-table-form.component.html

```diff
@@ -0,0 +1,85 @@
+<fieldset [formGroup]="form" class="lots-table">
```

Comment:
did you have any specific requirements to validations or just standard one from Material?

Reply:
no specific requirements, so for now its just the red outlines for required